src/panels/QueryEditorTab.ts

```diff
@@ -412,4 +437,41 @@ export class QueryEditorTab extends BaseTab {
         openSurvey(ExperienceKind.NoSQL, 'cosmosDB.nosql.queryEditor.provideFeedback');
         return Promise.resolve();
     }
+
+    private updateCsvSeparator() {
```

Comment:
Now that this lives in the host, there is no need for this, just pass the result from `SettingsService.getSetting` to the converter function as parameter, or even better, just let the converter function get the setting directly.

Reply:
Problem is that I always get doing that - so this is a work around:
![image](https://github.com/user-attachments/assets/c87e5030-0c9b-4ef6-a3b8-0fdd6677d49d)

Not sure why TBH - doesn't really make sense to me.

Reply:
@mkrueger Well the problem is that convertors.ts is now imported from both the WebView side and the host side, with the WebView side not being able to access vscode APIs. You can refactor the CSV converter in a separate csvConverter.ts so it won't get imported by the WebView code.